unified-runtime/source/adapters/level_zero/usm.cpp

```diff
@@ -1207,12 +1207,13 @@ ur_usm_pool_handle_t_::getPoolByHandle(const umf_memory_pool_handle_t UmfPool) {
 }
 
 void ur_usm_pool_handle_t_::cleanupPools() {
-  PoolManager.forEachPool([&](UsmPool *p) { return p->AsyncPool.cleanup(); });
+  PoolManager.forEachPool([&](UsmPool *p) { return p->AsyncPool.cleanup(); },
```

Comment:
Just change this function to always return true. No need to modify forEachPool.

Reply:
We rely on the return value of `cleanup` here https://github.com/intel/llvm/blob/sycl/unified-runtime/source/adapters/level_zero/usm.cpp#L1173.

Reply:
What I meant is:
```
PoolManager.forEachPool([&](UsmPool *p) { p->AsyncPool.cleanup(); return true; });
```

Reply:
Done.